File: app/src/main/java/lemond/annoying/gamerscompanion/game/objects/PEGI.java
package lemond.annoying.gamerscompanion.game.objects;


public class PEGI {

    public Rating rating;
    public String sypnosis;

    public enum Rating {
        //minimum age
        MIN3(1),
        MIN7(2),
        MIN12(3),
        MIN16(4),
        MIN18(5);

        private final int value;

        Rating(int value) {
            this.value = value;
        }

        public int getValue() {
            return value;
        }
    }
}

File: app/src/main/java/lemond/annoying/gamerscompanion/repository/objects/PEGI.kt
package lemond.annoying.gamerscompanion.repository.objects


class PEGI {

    var rating: Rating? = null
    var sypnosis: String? = null

    enum class Rating private constructor(val value: Int) {
        //minimum age
        MIN3(1),
        MIN7(2),
        MIN12(3),
        MIN16(4),
        MIN18(5)
    }
}
